openapi.yaml

```diff
@@ -8578,3 +8626,142 @@ components:
       - title
       - total_stats
       type: object
+    FilterDeploymentsRequest:
+      additionalProperties: false
+      description: Filters to select the deployments in `/filter/deployments`.
+      example:
+        account: 1
+        date_from: 2021-06-23
+        date_to: 2021-12-24
+        in:
+        - github.com/athenianco/athenian-api
+        - github.com/athenianco/athenian-webapp
+        - github.com/athenianco/metadata
+        environments:
+        - production
+        - staging
+        conclusions:
+        - FAILURE
+        with_labels:
+          user: github.com/lwsanty
+          rollback: null
+      properties:
+        date_from:
+          description: Date from when to search for the deployments,
```

Comment:
This aligns with the creation date of the deployment I suppose, not the releases, right?